tests/python/pytorch/graphbolt/test_base.py

```diff
@@ -248,6 +248,33 @@ def test_isin_non_1D_dim():
         gb.isin(elements, test_elements)
 
 
+def torch_expand_indptr(indptr, dtype, nodes=None):
+    if nodes is None:
+        nodes = torch.arange(len(indptr) - 1, dtype=dtype, device=indptr.device)
+    return nodes.to(dtype).repeat_interleave(indptr.diff())
+
+
+@pytest.mark.parametrize("nodes", [None, True])
+@pytest.mark.parametrize("dtype", [torch.int32, torch.int64])
+def test_expand_indptr(nodes, dtype):
+    num_nodes = 13
+    if nodes:
+        nodes = torch.randint(0, 199, [num_nodes], dtype=dtype, device=F.ctx())
+    for _ in range(10):
```

Comment:
why do we need for loop here?

Reply:
For better robustness, if the code has a bug, it will fail %99.999999 of the time with this many iterations. Using static test cases might not catch an existing bug if the bug still works with the hardcoded input test case.

Reply:
Mathematically speaking, such random test cases are more reliable than single hardcoded test cases. But if you insist, I can change to a hardcoded test case.

Reply:
This is not a good practise for unit test, design suitable test cases instead of rely on randomness.